src/MessageHandler/Consumer.php

```diff
@@ -67,6 +70,8 @@ public function getMessageEnvelope(): ?MessageEnvelope
             }
         }
 
+        $this->addLogOnRedisCall(__METHOD__, microtime(true) - $microtimeStart);
```

Comment:
I think this won't be call if a message envelope is returned (see return statement in this method)